app/Controllers/importExportController.php

```diff
@@ -256,11 +264,11 @@ private function ttrssXmlToJson(string $xml) {
 	/**
 	 * This method import a JSON-based file (Google Reader format).
 	 *
-	 * @param string $article_file the JSON file content.
-	 * @param boolean $starred true if articles from the file must be starred.
+	 * $article_file the JSON file content.
```

Comment:
Why did you remove `@param` ?

Reply:
As the function is typed and not to mislead the developer in case the documentation is not up-to-date. I could also have strange behaviors with phpstan